docs/io/fits/index.rst

```diff
@@ -530,6 +530,26 @@ encapsulate it in an :class:`HDUList` object first.
 Creating a New Table File
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. note::
+
+    If you want to create a simple binary FITS table with no other HDUs,
```

Comment:
Do skip on this, we shouldn't kill trees for changes like these (on the other hand I think binary FITS table is like a proper noun, never heard referring it in another way. But I'm sure there are weird FITS files out there that may fall into another category.)